### PR TITLE
Dev 1950 navigation updates for launch

### PR DESCRIPTION
**Pull request by gurjittoptal**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
new dashboard ui fixes.

#### What's this PR do?
handles multiple ui fixes for new dashboard

#### Why are we doing this? How does it help us?
for new design

#### Have automated unit tests been added?
n/a

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-1950
https://news-revenue-hub.atlassian.net/browse/DEV-1961: Align NRE logo in topbar for desktop
https://news-revenue-hub.atlassian.net/browse/DEV-1962 : remove dashboard link
https://news-revenue-hub.atlassian.net/browse/DEV-1966: show topbar in edit-page
https://news-revenue-hub.atlassian.net/browse/DEV-1991: nav icons swap and rotate

### CODE REVIEW

**Review by benjaminEwhite — APPROVED** [review]
One or two things that should be removed, but otherwise LGTM.

FWIW, I'm not on my normal computer, and wasn't able to run locally. I tried to access dashboard from review app, but got a 404 going to https://dev-1950.revengine-review.org/login . Doesn't seem like this should be caused by this PR, but also not sure why I wouldn't otherwise be able to access dashboard in review app.

File: spa/src/components/dashboard/sidebar/DashboardSidebar.js [hunk 1]
```diff
@@ -12,6 +12,15 @@ import {
 import flagIsActiveForUser from 'utilities/flagIsActiveForUser';
 import { useFeatureFlagsProviderContext } from 'components/Main';
 
+function DashboardMain() {
```
[the remark this answers is not in the document]
Comment (gurjittoptal) [reply spa/src/components/dashboard/sidebar/DashboardSidebar.js:17]: Not right now but will be in future, so i commented it out.

File: spa/src/components/dashboard/sidebar/DashboardSidebar.test.js [hunk 2]
```diff
@@ -16,16 +16,15 @@ jest.mock('utilities/flagIsActiveForUser', () => ({
 
 it('should have expected appearance and links', () => {
   render(<DashboardSidebar />);
-  const sidebar = screen.getByRole('list', { name: /Dashboard/ });
-
-  const contentNavSection = within(sidebar).getByRole('navigation', { name: /Content/ });
+  //const sidebar = screen.getByRole('list', { name: /Dashboard/ });
```
[the remark this answers is not in the document]
Comment (gurjittoptal) [reply spa/src/components/dashboard/sidebar/DashboardSidebar.test.js]: removed

**Comment by gurjittoptal** [comment]
re:I tried to access dashboard from review app, but got a 404 going to https://dev-1950.revengine-review.org/login

yeah, i also noticed that. Its should not be related to this PR.

problem was that while deploying review app 'DASHBOARD_SUBDOMAINS' was not set, ( none of ENVIRONMENT, DASHBOARD_SUBDOMAINS, SITE_URL). Will note this in DEV-1950 comments.

**Pull request merged**